crates/primitives/src/specification.rs

```diff
@@ -111,7 +106,7 @@ pub trait Spec: Sized {
             }
         }
 
-        Self::SPEC_ID as u8 >= spec_id as u8
+        SpecId::enabled(Self::SPEC_ID, spec_id)
```

Comment:
This is great, and you can even remove previous lines as they are checked inside `SpecId::enabled`

Reply:
Great finding!